### PR TITLE
V2 background color continuity across all pages #1200

### LINKED ISSUE #1200: v2 Background Color Continuity Across All Pages
### Overview
We need to match the background color (currently seen as black side panels on FAQ, Blog, Privacy, and Contact pages) to be the same as the background color on the Reports-Overview page so that the user experience is visually seamless (color inconsistencies) across all of the existing pages. 

Note: It looks like the Reports pages load in black first before rendering the final color. Is it possible to remove this color transition during rendering?

### Action Items
- [x] Identify Reports background color - Hex color: [0F181F] 
- [x] Apply to all pages 

### Resources/Instructions 
[Reports-Overview](https://dev.311-data.org/reports/dashboards/recent)
[FAQ](https://dev.311-data.org/faqs)

### PR DESCRIPTION
**Pull request by ardada2468**
Fixes #1200 

  - [x] Up to date with `dev` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [x] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: client/theme/theme.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const theme = createMuiTheme({
       light: colorTextSecondaryLight,
     },
     background: {
-      default: '#1A1A1A',
+      default: '#0f181f',
```
Comment (nichhk) [inline client/theme/theme.js:31]: It looks like we generally use uppercase for color codes, so let's stick with that for consistency.

**Review by nichhk — CHANGES_REQUESTED** [review]
Could you also share some screenshots showing the color change? Thanks!

**Comment by ardada2468** [comment]
image


This is how it looks on my end.

**Review by nichhk — APPROVED** [review]
LGTM! Thanks Arnav!

**Pull request merged**